src/OSPSuite.Infrastructure.Import/Core/DataSourceFileReaders/CsvDataSourceFile.cs

```diff
@@ -50,7 +51,8 @@ protected override void LoadFromFile(string path)
                while (csv.ReadNextRecord())
                {
                   csv.CopyCurrentRecordTo(currentRow);
-                  var rowList = currentRow.Select(x => x.Replace(csvSeparators.DecimalSeparator, '.')).ToList();
+                  var currentCultureDecimalSeparator = Convert.ToChar(Thread.CurrentThread.CurrentCulture.NumberFormat.NumberDecimalSeparator);
```

Comment:
This is pretty nice.
I pulled the branch and tried it on my PC and it works (using the starter project). However, it occurs to me that on my machine the new code is identical to what was there since my NumberDecimalSeparator will be '.' anyway.

Could you make sure that you can import this file from on your machine:
https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/issues/1821


You could probably also skip the Replace if the characters searched and replaced are the same anyway (ie both '.' or both ',')